ddtrace/internal/telemetry/writer.py

```diff
@@ -264,6 +265,7 @@ def __init__(self, is_periodic=True, agentless=None):
         self._configuration_queue = {}  # type: Dict[str, Dict]
         self._lock = forksafe.Lock()  # type: forksafe.ResetObject
         self._imported_dependencies: Dict[str, Distribution] = dict()
+        self._product_enablement = {product.value: False for product in TELEMETRY_APM_PRODUCT}
```

Comment:
With this approach could we report that a product is disabled when it is actually enabled?